homework-submission/react-hyf/react-hyf-client/src/components/Homeworkweek3/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import List from './List';
+
+class HomeworkWeek3 extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      isLoaded: false,
+      items: [],
+    };
+  }
+
+  componentDidMount() {
+    const handleErrors = response => {
+      if (!response.ok) {
+        throw Error(response.statusText);
+      }
+      return response;
+    };
+
+    fetch('https://uinames.com/api/?amount=10')
+      .then(handleErrors)
+      .then(res => res.json())
+      .then(items =>
+        this.setState(
+          {
+            isLoaded: true,
+            items,
+          },
+          () => console.log(items),
```

Comment:
Logs are good for development, but in a production this will affect user experience.